packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -1286,6 +1286,14 @@ const schema: SchemaType<DbUser> = {
     label: "Someone has mentioned me in a post or a comment",
     ...notificationTypeSettingsField(),
   },
+  notificationDebateCommentsOnSubscribedPost: {
+    label: "New debate content in a debate I'm subscribed to",
+    ...notificationTypeSettingsField({ batchingFrequency: 'daily' })
+  },
+  notificationDebateReplies: {
+    label: "New debate content in a debate I'm participating in",
```

Comment:
So... I don't _especially_ love this, but there are several problems & constraints this is working around:
1. Responses in debates are "comments"
2. Debate responses will have a very different pattern (incl. frequency) than regular comments on posts
3. Participants in a debate very likely want real-time (or close) notifications about other participants replying to the debate, while readers almost certainly don't want to get pinged ten times if two debate participants go back and forth five times in the space of 15-20 minutes
4. Both of these patterns are entirely separate from how both the participants and readers would likely want to experience being subscribed to non-participant comments on the debate, which should be much more like comments on other posts

I am going to ship this as-is since we have a rapidly approaching deadline but if there's a better solution I'm happy to discuss it.